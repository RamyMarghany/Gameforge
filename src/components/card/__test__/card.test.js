import React from 'react';
import ReactDOM from 'react-dom'
import Card from './../index'
import "@testing-library/jest-dom"; 
import renderer from 'react-test-renderer'

it('render without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Card></Card>, div)
})


it('matches snapshots', ()=>{
    const tree = renderer.create(<Card/>).toJSON()
    expect(tree).toMatchSnapshot();
})