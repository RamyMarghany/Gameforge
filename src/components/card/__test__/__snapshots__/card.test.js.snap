// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<div
  className="stylecss__CardContainer-sc-1ktc96q-0 dYBIaN"
  data-testid="card"
>
  <p
    className="stylecss__Info-sc-1ktc96q-1 ebXYvs"
  >
    <span
      className="stylecss__MainInfo-sc-1ktc96q-2 jkjNTl"
    >
       
    </span>
  </p>
  <p
    className="stylecss__Info-sc-1ktc96q-1 ebXYvs"
  >
    <span
      className="stylecss__MainInfo-sc-1ktc96q-2 jkjNTl"
    />
  </p>
</div>
`;
